public/app/features/alerting/unified/Templates.test.tsx

```diff
@@ -1,18 +1,22 @@
 import React from 'react';
-import { render, screen } from 'test/test-utils';
+import { render, screen, userEvent } from 'test/test-utils';
 
 import { setupMswServer } from 'app/features/alerting/unified/mockApi';
+import { setGrafanaAlertmanagerConfig } from 'app/features/alerting/unified/mocks/server/configure';
+import { AlertManagerCortexConfig } from 'app/plugins/datasource/alertmanager/types';
 import { AccessControlAction } from 'app/types';
 
 import Templates from './Templates';
-import setupGrafanaManagedServer from './components/contact-points/__mocks__/grafanaManagedServer';
 import { grantUserPermissions } from './mocks';
 
-const server = setupMswServer();
+jest.mock('app/core/components/AppChrome/AppChromeUpdate', () => ({
```

Comment:
(I tried to render `AppChrome` in the tests instead so it could show the actions on the page, but it wasn't playing ball. I had to pull in KBarProvider, and then the plugins hook was complaining, and then the tests started timing out... This pattern was used elsewhere and just makes it so the actions render on the page plainly)